client/src/components/AddCard/AddCardForm/CreditCard/CreditCard.tsx

```diff
@@ -0,0 +1,60 @@
+import useStyles from './useStyles';
+import { Typography } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import Radio from '@material-ui/core/Radio';
+import CheckIcon from '@material-ui/icons/Check';
+import { logos, logosModel } from './logos';
+import { theme } from '../../../../themes/theme';
+
+interface Props {
+  card: any;
+  setValues: (
+    values: React.SetStateAction<{
+      paymentMethod: string;
+    }>,
+    shouldValidate?: boolean | undefined,
+  ) => void;
+  handleSubmit: (e?: React.FormEvent<HTMLFormElement> | undefined) => void;
+  values: {
+    paymentMethod: string;
+  };
+}
+
+const CreditCard = ({ card, setValues, handleSubmit, values }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Box padding="20px" marginRight={2} border={2} borderColor="#dddddd" borderRadius={10} width={340} height={190}>
+      <Box height="36px" marginBottom={theme.spacing(0.4)} display="flex" justifyContent="space-between">
+        <img src={logos[card.card.brand as keyof logosModel]} className={classes.logo} />
```

Comment:
It is a good practice to add 'alt' attributes to HTML images elements (for accessibility purposes)

Reply:
Definitely. I missed it. Done.